bolt-contracts/config/holesky/operator.json

```diff
@@ -1,5 +1,6 @@
 {
-    "rpc": "localhost:50051",
-    "salt": "0x000000000000000abc0000000000000000000000000000000000000000000000",
-    "expiry": null
-}
\ No newline at end of file
+  "rpc": "<hostname>:<port>",
+  "salt": "0x0000000000000000000_salt_value_0000000000000000000000000000000000",
```

Comment:
This can be anything that is a bytes32 (so a hash or something), maybe we should add a section on what the salt does and how one can generate it using bash commands, options:
- `head -c 32 /dev/urandom | xxd -p -c 64`
- `openssl rand -hex 32`

(output will have to be 0x prefixed)